public/celestia-app.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+# ASCII art
+echo "          __        __  _                       "
+echo " _______ / /__ ___ / /_(_)__ ________ ____  ___ "
+echo "/ __/ -_) / -_|_-</ __/ / _ \`/___/ _ \`/ _ \/ _ \\"
+echo "\\__/\\__/_/\\__/___/\\__/_/\\_,_/    \\_,_/ .__/ .__/"
+echo "                                    /_/  /_/    "
+
+# Rest of your script...
+echo ""
+
+# Create a log file to capture detailed logs
+LOGFILE="$HOME/celestia-temp/logfile.log"
+TEMP_DIR="$HOME/celestia-temp"
+
+# Check if the directory exists
+if [ -d "$TEMP_DIR" ]; then
+    read -p "Directory $TEMP_DIR exists. Do you want to clear it out? (y/n) " -n 1 -r
```

Comment:
[not blocking] hmm I ran this script a few times and didn't encounter this message. 

Or perhaps I ran a previous version of this PR

Reply:
I'm sorry for tagging you for review too early! that's likely the case.